demo/home/home.html

```diff
@@ -2,7 +2,7 @@
 <div class="container">  
 	<div class="alert alert-warning" role="alert" ng-if="!isCachingAvailable">
 		<span class="glyphicon glyphicon-warning-sign" aria-hidden="true"></span>
-        The data cache is not available. Please start the <a href="#cache">caching process!!!</a>
+        The data cache is not available. Please start the <a href="#!cache">caching process!!!</a>
```

Comment:
Three exclamation marks might be a bit of overkill.

Reply:
It is an angry expression :-D, jokes apart I will remove it